api/api.go

```diff
@@ -77,3 +78,17 @@ func (binding *SplitioAPIBinding) GetSegmentChanges() error {
 func (binding *SplitioAPIBinding) GetSplitChanges() error {
 	return fmt.Errorf("not implemented")
 }
+
+func getSegmentNamesInUse(conditions []dtos.ConditionDTO) map[string]bool {
```

Comment:
I'm not sure how you expect to use this, but should it return an `[]string` instead?

Reply:
Yeah the use in `getSegmentsInUse` will use names exactly like a slice: 
1. Get the Len of names to calculate usingSegmentsCount
2. Loop through it to append it to `allSegmentNames`

Below is part of my code in getSegmentsForSplits function
```
	segmentNames := getSegmentNamesInUse(split.Conditions)
		if len(segmentNames) > 0 {
			usingSegmentsCount++
		}
		for segmentName := range segmentNames {
			allSegmentNames[segmentName] = true
		}
```

I think map can do the same thing so don't want to add extra codes to transfer a map to a slice in this function (?)